.github/workflows/download-crowdin.yml

```diff
@@ -0,0 +1,19 @@
+name: Crowdin  download Action
+
+on:
+  schedule:
+    - cron: "15 0 * * *"
```

Comment:
recommended to avoid the same cron for all download operations to prevent faced errors with crowd api (exceed of quotas builds)

Reply:
Done, Thanks.